tdx-tdcall/src/tdx.rs

```diff
@@ -37,6 +37,7 @@ pub struct TdInfo {
     pub attributes: u64,
     pub max_vcpus: u32,
     pub num_vcpus: u32,
+    pub vcpu_index: u32,
     pub rsvd: [u64; 3],
```

Comment:
If we add u32, we need reduce u32 from rsvd.

Reply:
yes, since `pub vcpu_index: u32` was added, to maintain the same struct size, `pub rsvd: [u64; 3]` was modified to `pub rsvd: [u32; 5]`.